src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -209,6 +211,32 @@ def subvolume_getpath(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def subvolume_info(self, **kwargs):
+        ret        = None
+        volname    = kwargs['vol_name']
+        subvolname = kwargs['sub_name']
+        groupname  = kwargs['group_name']
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                with open_group(fs_handle, self.volspec, groupname) as group:
+                    with open_subvol(fs_handle, self.volspec, group, subvolname) as subvolume:
+                        mon_addr_lst = []
+                        r, mon_map_json, err = self.mgr.mon_command({'prefix': 'mon dump', 'format': 'json'})
```

Comment:
instead of running a mon command, the `mon map` can be fetched by:

`self.mgr.get("mon_map")`

Reply:
Done